sentry/files/config.yml

```diff
@@ -0,0 +1,67 @@
+{%- from "sentry/map.jinja" import server with context %}
+# While a lot of configuration in Sentry can be changed via the UI, for all
+# new-style config (as of 8.0) you can also declare values here in this file
+# to enforce defaults or to ensure they cannot be changed via the UI. For more
+# information see the Sentry documentation.
+
+###############
+# Mail Server #
+###############
+
+{%- if server.mail.engine == 'smtp' %}
+mail.backend: 'smtp'  # Use dummy if you want to disable email entirely
+mail.host: '{{ server.mail.host }}'
+mail.port: {{ server.mail.port }}
+mail.username: '{{ server.mail.user }}'
+mail.password: '{{ server.mail.password }}'
+mail.use-tls: false
+# The email address to send on behalf of
+mail.from: 'sentry@localhost'
+{%- else %}
+mail.backend: 'dummy'
+{%- endif %}
+# If you'd like to configure email replies, enable this.
+# mail.enable-replies: false
+
+# When email-replies are enabled, this value is used in the Reply-To header
+# mail.reply-hostname: ''
+
+# If you're using mailgun for inbound mail, set your API key and configure a
+# route to forward to /api/hooks/mailgun/inbound/
+# mail.mailgun-api-key: ''
+
+###################
+# System Settings #
+###################
+
+# If this file ever becomes compromised, it's important to regenerate your a new key
+# Changing this value will result in all current sessions being invalidated.
+# A new key can be generated with `$ sentry config generate-secret-key`
+system.secret-key: '{{ server.secret_key }}'
+
+# The ``redis.clusters`` setting is used, unsurprisingly, to configure Redis
+# clusters. These clusters can be then referred to by name when configuring
+# backends such as the cache, digests, or TSDB backend.
+redis.clusters:
+  default:
+    hosts:
+      0:
```

Comment:
It would be good to have support for multiple redis hosts in metadata model as it seems that sentry supports it.